go.mod

```diff
@@ -125,16 +124,17 @@ require (
 	github.com/vishvananda/netns v0.0.0-20171111001504-be1fbeda1936 // indirect
 	github.com/vmware/govmomi v0.20.1
 	github.com/xlab/handysort v0.0.0-20150421192137-fb3537ed64a1 // indirect
+	go.etcd.io/etcd v0.0.0-20191023171146-3cf2f69b5738
 	golang.org/x/crypto v0.0.0-20190611184440-5c40567a22f8
 	golang.org/x/lint v0.0.0-20190409202823-959b441ac422
-	golang.org/x/net v0.0.0-20190812203447-cdfb69ac37fc
+	golang.org/x/net v0.0.0-20190813141303-74dc4d7220e7
```

Comment:
@jpbetz what version of go is etcd 3.4.3 building with? this is the go1.13 version of golang.org/x/net, but we are still pinned to the 1.12 version (until https://github.com/kubernetes/kubernetes/issues/82531 lands)

Reply:
etcd v3.4.3 is built with go version 1.12.12 https://github.com/etcd-io/etcd/blob/v3.4.3/Makefile#L54. What shall I do this golang.org/x/net  dependency then?

Reply:
we're still pinned to the go1.12 version of golang.org/x/net with our replace statement, but this means etcd is building/testing with a mix of a go1.12 runtime and a go1.13-level golang.org/x/net library (which @jpbetz might want to reconcile)